### ISSUE TITLE
Support seccomp profile as part of a pod/deployment security_context spec

### ISSUE THREAD

**Issue by tareks** [hashicorp/terraform-provider-kubernetes#1624]
### Description

It appears that the `secCompProfile` attribute that can be defined as part of a `securityContext` for a pod spec is not supported. 

Example yaml:
```
apiVersion: v1
kind: Pod
metadata:
  name: fine-pod
  labels:
    app: fine-pod
spec:
  securityContext:
    seccompProfile:
      type: Localhost
      localhostProfile: profiles/fine-grained.json
```

This would also apply to `kubernetes_deployment`, `kubernetes_job`, `kubernetes_statefulset`, and possibly others including `init_container` sections within some resources.

### Potential Terraform Configuration

```hcl
      spec {
        security_context {
          seccomp_profile {
            type = "Localhost"
            localhost_profile = "profiles/fine-grained.json"
          }
        }

```

### References

See: https://kubernetes.io/docs/tutorials/security/seccomp/

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Comment by arybolovlev** [comment]
Hi @tareks,

Thank you for reporting this issue. It seems to be a duplicate of this [one](https://github.com/hashicorp/terraform-provider-kubernetes/issues/1604). [PR](https://github.com/hashicorp/terraform-provider-kubernetes/pull/1617) is ready and waiting for review. Once it is approved and merged, the changes will be available in the upcoming release. Please keep an eye on release notes.

Please vote on the original issue by adding a 👍 reaction.

Thank you!
